src/pages/NotFound.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+const NotFound = () => {
+  let randomNumber = Math.ceil(Math.random()*3)
+  const Message = () => {
```

Comment:
It looks like Message is just a regular javascript function which means it should be `message`